src/index.ts

```diff
@@ -44,6 +45,10 @@ catch {
   output.push({ info: `👉 Using default configuration` })
 }
 
+// Check if `apply` or `ignore` options were explicitly passed
+const applyFromCLI = wasOptionPassed('apply')
```

Comment:
What about when both apply and ignore are present in the config file?
Or apply in the config and ignore from cli?

Should not we fire this in all cases?

Reply:
On it, I'll add more tests to check the behavior in those cases 👀